xarray/conventions.py

```diff
@@ -32,6 +44,18 @@
 )
 
 
+if TYPE_CHECKING:
+    from xarray.backends.common import AbstractDataStore
+    from xarray.core.dataset import Dataset
+
+    T_VarTuple = Tuple[Tuple[Hashable, ...], Any, dict, dict]
+    T_Name = Union[Hashable, None]
+    T_Variables = Mapping[Any, Variable]
+    T_Attrs = MutableMapping[Any, Any]
+    T_DropVariables = Union[str, Iterable[str], None]
```

Comment:
I think that should be `Union[str, Iterable[Hashable], None]`

Reply:
Done.